CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.35.1](https://github.com/buildkite/agent/tree/v3.35.1) (2022-04-05)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.35.0...v3.35.1)
+
+### Fixed
+
+- Revet file permissioning changes made in [#1580](https://github.com/buildkite/agent/pull/1580). They were creating issues with docker-based workflows [#1601](https://github.com/buildkite/agent/pull/1601) (@pda + @moskyb)
```

Comment:
Tiny typo in “revert”.
(I'm willing to be convinced that permissioning is a word if you're keen, though 😁)

```suggestion
- Revert file permission changes made in [#1580](https://github.com/buildkite/agent/pull/1580). They were creating issues with docker-based workflows [#1601](https://github.com/buildkite/agent/pull/1601) (@pda + @moskyb)
```

Reply:
😱 is permissioning not a word????